tests/Config/LoaderTest.php

```diff
@@ -1,11 +1,13 @@
 <?php
 
+namespace Notifly\Tests;
+
 /**
  * Class LoaderTest
  *
  * @covers \Notifly\Config\Loader
  */
-class LoaderTest extends PHPUnit_Framework_TestCase
+class LoaderTest extends \PHPUnit_Framework_TestCase
```

Comment:
Is this the only test that you are prefixing the extends class with a `\`?
Feel like it should be done across the board or not at all.